README.adoc

```diff
@@ -69,25 +69,25 @@ You see an output similar to the following example:
 ----
 appsody repo list
 NAME        URL
-*appsodyhub https://github.com/appsody/stacks/releases/latest/download/incubator-index.yaml
+*incubator https://github.com/appsody/stacks/releases/latest/download/incubator-index.yaml
 ----
 
 Next, run the following command to add the Kabanero index:
 [role="command"]
 ----
-appsody repo add kabanero https://github.com/kabanero-io/collections/releases/download/v0.1.2/kabanero-index.yaml
+appsody repo add kabanero https://github.com/kabanero-io/collections/releases/download/v0.2.1/kabanero-index.yaml
```

Comment:
this URL returns a 404 (there is no `v` in front of 0.2.0

I would suggest just making this general and point to the latest Collections release such as:
``appsody repo add kabanero-latest https://github.com/kabanero-io/collections/releases/latest/download/kabanero-index.yaml`

I am unsure someone would want to do this (does it pull the latest every time, or is it just set at whatever is the latest at the time of repo add)

if we can do this, you won't have to keep updating this guide every release

Reply:
@kilnerm - what do you recommend? ^^

Reply:
@SueChaplain I don't see a problem with updating to latest. We should probably add a checkpoint before each release to ensure the guide content is not regressed but other than that it makes sense to have people pick up the latest versions.

Reply:
@alohr51 - the "v" got removed in an earlier release. I'll update this PR. Note that https://kabanero.io/developer/cli/ has the same error and also needs an update.